sunpy/map/tests/test_reproject_to.py

```diff
@@ -139,3 +140,37 @@ def test_reproject_to_warn_using_contexts(aia171_test_map, hpc_header):
             # Check if a warning is raised if both context managers are used at the same time.
             with pytest.warns(UserWarning, match="Using propagate_with_solar_surface and SphericalScreen together result in loss of off-disk data."):
                 aia171_test_map.reproject_to(hpc_header)
+
+
+@figure_test
+@pytest.mark.parametrize('auto_extent', [None, 'corners', 'edges', 'all'])
+def test_reproject_to_auto_extent(aia171_test_map, auto_extent):
```

Comment:
Could we not write a numerical test with a known WCS rather than just a figure test?

Reply:
Done